.github/workflows/unit_test_bcc.yml

```diff
@@ -5,8 +5,10 @@ on:
 
 jobs:
   unit_test_with_bcc:
-    runs-on: ubuntu-latest
+    runs-on: c
     steps:
     - uses: actions/checkout@v3
+    - name: install bcc
```

Comment:
if you want to install bcc,  you can ref https://github.com/sustainable-computing-io/kepler/blob/37996282db229b8ce7c0433a856721b2c1a56400/.github/workflows/unit_test.yml#L29-L33
as apt repo's bcc is out of date.
or you just want header?

Reply:
right, we probably don't need the bcc.

@SamYuan1990 I updated the PR, can you review again?

Reply:
I am not sure for either #642 or this one works.
if #642 works, which means we can remove one test pipeline as duplication. @rootfs